BaseModel/src/com/wpi/cs509/teamA/bean/Node.java

```diff
@@ -71,6 +71,11 @@ public Node(int id, String name, int x, int y, int mapId, Set<Integer> neighbors
 
 	}
 
+	public Node(int id)
```

Comment:
Can you make a comment that this is just for test? We may not need this in the future.